algoliasearch_django/index/index.py

```diff
@@ -0,0 +1,164 @@
+from __future__ import unicode_literals
+
+import inspect
+from functools import partial
+from itertools import chain
+import logging
+import sys
```

Comment:
Not sure its something that should be done within this PR, but I can't go past poorly sorted imports! 8)

Reply:
I'd like to keep the diff between upstream and our fork as minimal as possible, so I don't want to make stylistic changes at this point. :)